deploy/role.yaml

```diff
@@ -44,6 +44,18 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
```

Comment:
Can we remove the deploymentconfig permissions now? I can't pick the line in this pr. It's line 74 in this file

Reply:
We cannot delete that permission, since the migration code also requires to list,get,delete the DCs. we can get rid of some of the perrmissions (u

Reply:
Yep, makes sense. I guess we can remove it when we are finished with the upgrade logic in a few versions. Resolving this conversation.